src/test/java/ca/team2706/frc/robot/commands/drivebase/TankDriveTest.java

```diff
@@ -0,0 +1,111 @@
+package ca.team2706.frc.robot.commands.drivebase;
+
+import com.ctre.phoenix.CTREJNIWrapper;
+import com.ctre.phoenix.motorcontrol.SensorCollection;
+import com.ctre.phoenix.motorcontrol.can.MotControllerJNI;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.sensors.PigeonIMU;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.function.Supplier;
+
+import ca.team2706.frc.robot.subsystems.DriveBase;
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.Notifier;
+import edu.wpi.first.wpilibj.PWM;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import mockit.Expectations;
+import mockit.Injectable;
+import mockit.Mocked;
+import mockit.Verifications;
+import util.Util;
+
+public class TankDriveTest {
+    @Mocked
+    private Supplier<Double> leftSpeed;
```

Comment:
Better to use `@Injectable` here instead of `@Mocked` since you want to run the verifications on the specific instance.